CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.17)
```

Comment:
Updated with an explicit warning message but just let me know if that's not desired.  Had to bump the CMake version to be able to get the needed flag.  Also happy to open an issue because that was going to be a part of what I was going to look into next along with looking into a container to build SIMA.

Reply:
You can see an example of what the warning message looks like here:
https://github.com/mwaxmonsky/history_output/actions/runs/10738355210/job/29781759502#step:4:15

Reply:
That warning looks good to me!  And definitely feel free to open an issue whenever you get the chance (that way we'll have a record of at least one next step in case you get pulled off onto something else).  Thanks!